### ISSUE TITLE
Add an environment variable to enable markdown

### ISSUE THREAD

**Issue by dfyx** [spruce/hubot-gitlab-hooks#10]
Kind of related to #9

It would be really cool to have an environment variable like `FORMATTING=markdown` to enable markdown in messages.

Some ideas what could be done with that:

* Link user names to their profiles
* Inline links that are currently in parentheses
* Link project names

**Comment by dfyx** [comment]
A simple way to handle this would be to always write markdown and use [summarize-markdown](https://www.npmjs.com/package/summarize-markdown) to strip out all markdown when none is supported.

**Comment by spruce** [comment]
Good idea. the problem with that library would be that it completly strips the linktarget from links 
https://github.com/bryanburgers/summarize-markdown/blob/master/index.js#L55-L57

**Comment by dfyx** [comment]
I'd say for some links (such as links to user profiles or even individual commits) that's the expected behavior but I agree that for some links, the target url should stay in the message.

The alternative would be to create dedicated renderer objects for every format.

**Comment by dfyx** [comment]
Also, some alternatives:

* [strip-markdown](https://github.com/wooorm/strip-markdown)
* [remove-markdown](https://github.com/stiang/remove-markdown)
